docs/contributing/building.rst

```diff
@@ -6,7 +6,7 @@ Building
 
 .. only:: internal
 
-   If you are building katana-enterprise make sure to also read the :doc:`enterprise build addenda <index>`.
+   If you are building katana-enterprise make sure to also read the :ref:`enterprise-building`.
```

Comment:
Please create a building-internal.rst file on the enterprise side that has this using include directive. There are a number of ways to accomplish this effect. If you'd like some advice on how to do this, reach out to me.